numpy/core/src/multiarray/nditer_pywrap.c

```diff
@@ -1065,6 +1088,12 @@ NpyIter_NestedIters(PyObject *NPY_UNUSED(self),
             Py_DECREF(ret);
             goto fail;
         }
+        if ((iter->has_writebackifcopy != 0) &&
+            (iter->managed != CONTEXT_INSIDE)) {
```

Comment:
This second condition is always `true`, right, because there's no way to be inside the context already here?

Reply:
gaah, this is a bug. I need to check the operands and set ``iter->has_writebackifcopy`` appropriately